--- djvulibre-3.5.18/desktopfiles/djvulibre-djview3.desktop.orig	2007-01-15 05:19:46.000000000 +0100
+++ djvulibre-3.5.18/desktopfiles/djvulibre-djview3.desktop	2007-02-04 21:06:05.879093998 +0100
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Type=Application
-Categories=Qt;Graphics
+Categories=Qt;Graphics;Viewer;
 MimeType=image/x-djvu;image/x.djvu;image/vnd.djvu;
 Name=DjView3
 Exec=djview3 %f
--- djvulibre-3.5.19/desktopfiles/Makefile.in.orig	2007-03-25 22:48:22.000000000 +0200
+++ djvulibre-3.5.19/desktopfiles/Makefile.in	2007-08-30 18:58:54.537469369 +0200
@@ -38,6 +38,9 @@
 datadir_djvu = ${datadir}/djvu/osi/desktop
 datadir_djview = ${datadir}/djvu/djview3/desktop
 
+desktopdir = /usr/share/applications
+iconsdir = /usr/share/icons
+mimedir = /usr/share/mime
 
 # -------- all
 
@@ -70,20 +73,16 @@
 @GUI_YES@install: install-djview-files
 
 install-djvu-files: FORCE
-	${INSTALL} -d ${DESTDIR}${datadir_djvu}
-	${INSTALL_PROGRAM} register-djvu-mime ${DESTDIR}${datadir_djvu}
-	${INSTALL_DATA} ${srcdir}/*djvu.png ${DESTDIR}${datadir_djvu}
-	${INSTALL_DATA} ${srcdir}/*mime.xml ${DESTDIR}${datadir_djvu}
-	-@echo "Run ${datadir_djvu}/register-djvu-mime install"
-	-@echo " to register the djvu mime types and icons."
+	${INSTALL} -d ${DESTDIR}${mimedir}/packages ${DESTDIR}${iconsdir}/hicolor/{22x22,32x32,48x48}/{mimetypes,apps}
+	${INSTALL_DATA} ${srcdir}/hi22-djvu.png ${DESTDIR}${iconsdir}/hicolor/22x22/mimetypes/mime-image-vnd.djvu.png
+	${INSTALL_DATA} ${srcdir}/hi32-djvu.png ${DESTDIR}${iconsdir}/hicolor/32x32/mimetypes/mime-image-vnd.djvu.png
+	${INSTALL_DATA} ${srcdir}/hi48-djvu.png ${DESTDIR}${iconsdir}/hicolor/48x48/mimetypes/mime-image-vnd.djvu.png
+	#${INSTALL_DATA} ${srcdir}/djvulibre-mime.xml ${DESTDIR}${mimedir}/packages/djvulibre.xml
 
 install-djview-files: FORCE
-	${INSTALL} -d ${DESTDIR}${datadir_djview}
-	${INSTALL_PROGRAM} register-djview-menu ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} ${srcdir}/*djview3.png ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} ${srcdir}/*djview3.desktop ${DESTDIR}${datadir_djview}
-	-@echo "Run ${datadir_djview}/register-djview-menu install"
-	-@echo " to register the djview3 menu entries."
+	${INSTALL} -d ${DESTDIR}${desktopdir} ${DESTDIR}${iconsdir}/hicolor/{22x22,32x32,48x48}/{mimetypes,apps}
+	${INSTALL_DATA} ${srcdir}/hi32-djview3.png ${DESTDIR}${iconsdir}/hicolor/32x32/apps/djvulibre-djview3.png
+	${INSTALL_DATA} ${srcdir}/djvulibre-djview3.desktop ${DESTDIR}${desktopdir}/djvulibre-djview3.desktop
 
 
 # -------- misc
