--- djvulibre-3.5.14/gui/desktop/djview.desktop.orig	2004-05-05 18:10:01.000000000 +0200
+++ djvulibre-3.5.14/gui/desktop/djview.desktop	2004-07-25 21:04:09.754254528 +0200
@@ -11,6 +11,8 @@
 GenericName[fr]=Afficheur DjVu
 GenericName[it]=Visualizzatore DjVu
 GenericName[ja]=DjVuビューア
+GenericName[pl]=Przeglądarka DjVu
 StartupWMClass=Djview
-Terminal=0
-Categories=Qt;Graphics
+Terminal=false
+Categories=Qt;Graphics;Viewer;
+# vi: encoding=utf-8
