--- ./gui/nsdejavu/Makefile.in.org	Wed Jan 16 20:52:38 2002
+++ ./gui/nsdejavu/Makefile.in	Thu May 16 23:24:55 2002
@@ -67,12 +67,12 @@
 install: install-bin install-man
 
 install-bin: ${NSDEJAVU} FORCE
-	${INSTALL} -d ${plugindir}
-	${INSTALL} -m 0644 ${NSDEJAVU} ${plugindir}
+	${INSTALL} -d ${DESTDIR}${plugindir}
+	${INSTALL} -m 0644 ${NSDEJAVU} ${DESTDIR}${plugindir}
 
 install-man: nsdejavu.1 FORCE
-	${INSTALL} -d ${mandir}/man1
-	${INSTALL} -m 0644 nsdejavu.1 ${mandir}/man1
+	${INSTALL} -d ${DESTDIR}${mandir}/man1
+	${INSTALL} -m 0644 nsdejavu.1 ${DESTDIR}${mandir}/man1
 
 depend: FORCE
 	${CC} -MM ${CFLAGS} ${srcdir}/nsdejavu.c > Makefile.dep
--- ./gui/djview/Makefile.in.org	Thu May 16 23:24:09 2002
+++ ./gui/djview/Makefile.in	Thu May 16 23:24:00 2002
@@ -180,19 +180,19 @@
 install: install-bin install-man install-data
 
 install-bin: ${PROGRAM} FORCE
-	${INSTALL} -d ${bindir}
-	${INSTALL} -s -m 0755 ${PROGRAM} ${bindir}
+	${INSTALL} -d ${DESTDIR}${bindir}
+	${INSTALL} -s -m 0755 ${PROGRAM} ${DESTDIR}${bindir}
 
 install-man: FORCE
-	${INSTALL} -d ${mandir}/man1
-	${INSTALL} -m 0644 ${srcdir}/djview.1 ${mandir}/man1
+	${INSTALL} -d ${DESTDIR}${mandir}/man1
+	${INSTALL} -m 0644 ${srcdir}/djview.1 ${DESTDIR}${mandir}/man1
 
 install-data: FORCE
-	${INSTALL} -d ${datadir}/djvu/osi
+	${INSTALL} -d ${DESTDIR}${datadir}/djvu/osi
 	-test -r ${srcdir}/djview_help.djvu && \
-	  ${INSTALL} -m 0644 ${srcdir}/djview_help.djvu ${datadir}/djvu/osi
+	  ${INSTALL} -m 0644 ${srcdir}/djview_help.djvu ${DESTDIR}${datadir}/djvu/osi
 	-test -r ${srcdir}/djview_help.html && \
-	  ${INSTALL} -m 0644 ${srcdir}/djview_help.html ${datadir}/djvu/osi
+	  ${INSTALL} -m 0644 ${srcdir}/djview_help.html ${DESTDIR}${datadir}/djvu/osi
 
 
 depend: ${MOCINCS} FORCE
--- ./libdjvu/Makefile.in.org	Thu May 16 23:15:27 2002
+++ ./libdjvu/Makefile.in	Thu May 16 23:17:47 2002
@@ -99,15 +99,15 @@
 @RPO_YES@	${CXX} ${CXXFLAGS} -o $@ $@.o ${OBJS} ${LIBS} 
 
 install-data: FORCE
-	${INSTALL} -d ${datadir}/djvu
+	${INSTALL} -d ${DESTDIR}${datadir}/djvu
 	for n in `cd ${top_srcdir}/share && find djvu -name '*.xml' -print` ; do \
 	  src="${top_srcdir}/share/$$n" ; dir="`dirname ${datadir}/$$n`" ;\
-	  test -d $$dir || ${INSTALL} -d $$dir ; \
-	  ${INSTALL} -m 0644 $$src $$dir || exit ; done
+	  test -d ${DESTDIR}$$dir || ${INSTALL} -d ${DESTDIR}$$dir ; \
+	  ${INSTALL} -m 0644 $$src ${DESTDIR}$$dir || exit ; done
 
 install-lib: ${LIBDJVU_SO} FORCE
-	${INSTALL} -d ${libdir}
-	${INSTALL} -m 644 ${LIBDJVU_SO} ${libdir}
+	${INSTALL} -d ${DESTDIR}${libdir}
+	${INSTALL} -m 644 ${LIBDJVU_SO} ${DESTDIR}${libdir}
 
 depend: FORCE
 	${CXX} -MM ${CXXFLAGS} ${srcdir}/*.cpp > Makefile.dep
--- ./xmltools/Makefile.in.org	Thu May 16 23:19:47 2002
+++ ./xmltools/Makefile.in	Thu May 16 23:21:26 2002
@@ -63,21 +63,21 @@
 install: install-bin install-data # install-man (disabled for now)
 
 install-bin: ${PROGRAMS} FORCE
-	${INSTALL} -d ${bindir}
+	${INSTALL} -d ${DESTDIR}${bindir}
 	for n in ${PROGRAMS} ; do \
-	  ${INSTALL} -s -m 0755 $$n ${bindir} || exit ; done
+	  ${INSTALL} -s -m 0755 $$n ${DESTDIR}${bindir} || exit ; done
 
 install-data: FORCE
-	${INSTALL} -d ${datadir}/djvu
+	${INSTALL} -d ${DESTDIR}${datadir}/djvu
 	for n in `cd ${top_srcdir}/share && find djvu -name '*.dtd' -print` ; do \
 	  src="${top_srcdir}/share/$$n" ; dir="`dirname ${datadir}/$$n`" ;\
-	  test -d $$dir || ${INSTALL} -d $$dir ; \
-	  ${INSTALL} -m 0644 $$src $$dir || exit ; done
+	  test -d ${DESTDIR}$$dir || ${INSTALL} -d ${DESTDIR}$$dir ; \
+	  ${INSTALL} -m 0644 $$src ${DESTDIR}$$dir || exit ; done
 
 install-man: FORCE
-	${INSTALL} -d ${mandir}/man1
+	${INSTALL} -d ${DESTDIR}${mandir}/man1
 	for n in ${srcdir}/*.1 ; do \
-	  ${INSTALL} -m 0644 $$n ${mandir}/man1 || exit ; done 
+	  ${INSTALL} -m 0644 $$n ${DESTDIR}${mandir}/man1 || exit ; done 
 
 depend: FORCE
 	${CXX} -MM ${CXXFLAGS} ${srcdir}/*.cpp > Makefile.dep
--- ./tools/Makefile.in.org	Thu May 16 23:19:02 2002
+++ ./tools/Makefile.in	Thu May 16 23:18:49 2002
@@ -65,14 +65,14 @@
 install: install-bin install-man
 
 install-bin: ${PROGRAMS} FORCE
-	${INSTALL} -d ${bindir}
+	${INSTALL} -d ${DESTDIR}${bindir}
 	for n in ${PROGRAMS} ; do \
-	  ${INSTALL} -s -m 0755 $$n ${bindir} || exit ; done
+	  ${INSTALL} -s -m 0755 $$n ${DESTDIR}${bindir} || exit ; done
 
 install-man: FORCE
-	${INSTALL} -d ${mandir}/man1
+	${INSTALL} -d ${DESTDIR}${mandir}/man1
 	for n in ${srcdir}/*.1 ; do \
-	  ${INSTALL} -m 0644 $$n ${mandir}/man1 || exit ; done 
+	  ${INSTALL} -m 0644 $$n ${DESTDIR}${mandir}/man1 || exit ; done 
 
 depend: FORCE
 	${CXX} -MM ${CXXFLAGS} ${srcdir}/*.cpp > Makefile.dep
