--- djvulibre-3.5.18/gui/djview/Makefile.in.orig	2007-01-08 15:33:17.000000000 +0100
+++ djvulibre-3.5.18/gui/djview/Makefile.in	2007-02-04 21:04:26.349422126 +0100
@@ -174,7 +174,7 @@
 install-bin: ${PROGRAM} FORCE
 	${INSTALL} -d ${DESTDIR}${bindir}
 	${LIBTOOL} --mode=install \
-	  ${INSTALL_PROGRAM} -s ${PROGRAM} ${DESTDIR}${bindir}/djview3
+	  ${INSTALL_PROGRAM} ${PROGRAM} ${DESTDIR}${bindir}/djview3
 	cd ${DESTDIR}${bindir} && ${LN_S} djview3 djview || true
 
 install-man: FORCE
diff -aurN djvulibre-3.5.15.orig/tools/Makefile.in djvulibre-3.5.15/tools/Makefile.in
--- djvulibre-3.5.15.orig/tools/Makefile.in	2005-03-13 20:17:32.000000000 +0100
+++ djvulibre-3.5.15/tools/Makefile.in	2005-07-19 19:15:38.000000000 +0200
@@ -71,7 +71,7 @@
 	${INSTALL} -d ${DESTDIR}${bindir}
 	for n in ${PROGRAMS} ; do \
 	  ${LIBTOOL} --mode=install \
-	    ${INSTALL_PROGRAM} -s $$n ${DESTDIR}${bindir} \
+	    ${INSTALL_PROGRAM} $$n ${DESTDIR}${bindir} \
           || exit ; done
 
 install-man: FORCE
