--- djvulibre-3.5.18/gui/nsdejavu/Makefile.in.orig	2004-08-24 19:58:55.000000000 +0200
+++ djvulibre-3.5.18/gui/nsdejavu/Makefile.in	2007-02-04 23:59:24.915701322 +0100
@@ -51,7 +51,7 @@
 
 CFLAGS = @CPPFLAGS@ ${DEFS} ${OPTS} ${INCS} ${NSDEJAVU_CFLAGS} 
 
-LIBS = @LDFLAGS@ ${NSDEJAVU_LIBS} 
+LIBS = @LDFLAGS@ ${NSDEJAVU_LIBS} -lXt -lX11
 
 all: nsdejavu.la nsdejavu.x nsdejavu.1
 
--- djvulibre-3.5.23/gui/nsdejavu/Makefile.in~	2010-11-14 16:42:34.754225193 +0100
+++ djvulibre-3.5.23/gui/nsdejavu/Makefile.in	2010-11-14 16:42:44.722533345 +0100
@@ -62,7 +62,7 @@
 
 nsdejavu.la: nsdejavu.lo
 	${LIBTOOL} --mode=link ${CC} ${CFLAGS} nsdejavu.lo ${LIBS} \
-	  -rpath ${plugindir} -module -o $@
+	  -rpath ${plugindir} -module -avoid-version -o $@
 
 nsdejavu.x: nsdejavu.la FORCE
 	eval `grep '^dlname=' nsdejavu.la` && \
