--- djvulibre-3.5.18/desktopfiles/djvulibre-djview3.desktop.orig	2007-01-15 05:19:46.000000000 +0100
+++ djvulibre-3.5.18/desktopfiles/djvulibre-djview3.desktop	2007-02-04 21:06:05.879093998 +0100
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Type=Application
-Categories=Qt;Graphics
-MimeType=image/x-djvu;image/x.djvu;image/vnd.djvu
+Categories=Qt;Graphics;Viewer;
+MimeType=image/x-djvu;image/x.djvu;image/vnd.djvu;
 Name=DjView3
 Exec=djview3 %f
 Terminal=false
--- djvulibre-3.5.18/desktopfiles/Makefile.in.orig	2007-01-14 19:26:33.000000000 +0100
+++ djvulibre-3.5.18/desktopfiles/Makefile.in	2007-02-04 21:42:41.916239023 +0100
@@ -37,6 +37,9 @@
 datadir_djvu = ${datadir}/djvu/osi/desktop
 datadir_djview = ${datadir}/djvu/djview3/desktop
 
+desktopdir = /usr/share/applications
+iconsdir = /usr/share/icons
+mimedir = /usr/share/mime
 
 # -------- all
 
@@ -81,16 +84,16 @@
 @GUI_YES@echo-xdg: echo-djview-xdg
 
 install-djvu-files: FORCE
-	${INSTALL} -d ${DESTDIR}${datadir_djvu}
-	${INSTALL_PROGRAM} register-djvu-mime ${DESTDIR}${datadir_djvu}
-	${INSTALL_DATA} ${srcdir}/*djvu.png ${DESTDIR}${datadir_djvu}
-	${INSTALL_DATA} ${srcdir}/*mime.xml ${DESTDIR}${datadir_djvu}
+	${INSTALL} -d ${DESTDIR}${mimedir}/packages ${DESTDIR}${iconsdir}/hicolor/{22x22,32x32,48x48}/{mimetypes,apps}
+	${INSTALL_DATA} ${srcdir}/hi22-djvu.png ${DESTDIR}${iconsdir}/hicolor/22x22/mimetypes/djvu.png
+	${INSTALL_DATA} ${srcdir}/hi32-djvu.png ${DESTDIR}${iconsdir}/hicolor/32x32/mimetypes/djvu.png
+	${INSTALL_DATA} ${srcdir}/hi48-djvu.png ${DESTDIR}${iconsdir}/hicolor/48x48/mimetypes/djvu.png
+	#${INSTALL_DATA} ${srcdir}/djvulibre-mime.xml ${DESTDIR}${mimedir}/packages/djvulibre.xml
 
 install-djview-files: FORCE
-	${INSTALL} -d ${DESTDIR}${datadir_djview}
-	${INSTALL_PROGRAM} register-djview-menu ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} ${srcdir}/*djview3.png ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} ${srcdir}/*djview3.desktop ${DESTDIR}${datadir_djview}
+	${INSTALL} -d ${DESTDIR}${desktopdir} ${DESTDIR}${iconsdir}/hicolor/{22x22,32x32,48x48}/{mimetypes,apps}
+	${INSTALL_DATA} ${srcdir}/hi32-djview3.png ${DESTDIR}${iconsdir}/hicolor/32x32/apps/djview3.png
+	${INSTALL_DATA} ${srcdir}/djvulibre-djview3.desktop ${DESTDIR}${desktopdir}/djview3.desktop
 
 run-djvu-xdg: FORCE
 	-test -z "${DESTDIR}" && \
