--- djvulibre-3.5.27/config/acinclude.m4.orig	2016-01-09 21:49:51.852007828 +0100
+++ djvulibre-3.5.27/config/acinclude.m4	2016-01-09 22:38:16.038552615 +0100
@@ -90,31 +90,8 @@
         [ac_debug=$enableval],[ac_debug=no])
    defines=
    if test x$ac_debug = xno ; then
-     AC_REMOVE_OPTIONS([CFLAGS],[-O*])
-     AC_REMOVE_OPTIONS([CXXFLAGS],[-O*])
-     if test x$GCC != xyes ; then
-       AC_REMOVE_OPTIONS([CFLAGS],[-g*])
-       AC_REMOVE_OPTIONS([CXXFLAGS],[-g*])
-     fi
      defines="-DNDEBUG"
-     AC_CHECK_CC_OPT([-O3],,[AC_CHECK_CC_OPT([-O2])])
-     AC_CHECK_CXX_OPT([-O3],,[AC_CHECK_CXX_OPT([-O2])])
-     cpu=`uname -m 2>/dev/null`
-     test -z "$cpu" && cpu=${host_cpu}
-     case "${host_cpu}" in
-        i?86)
-           opt="-mtune=${host_cpu}"
-           AC_CHECK_CXX_OPT([-mtune=${host_cpu}],,
-                [AC_CHECK_CXX_OPT([-mcpu=${host_cpu}])])
-           AC_CHECK_CC_OPT([-mtune=${host_cpu}],,
-                [AC_CHECK_CC_OPT([-mcpu=${host_cpu}])])
-           ;;
-      esac
    else
-     AC_REMOVE_OPTIONS([CFLAGS],[-O*|-g*])
-     AC_REMOVE_OPTIONS([CXXFLAGS],[-O*|-g*])
-     AC_CHECK_CC_OPT([-g])
-     AC_CHECK_CXX_OPT([-g])
      AC_CHECK_CXX_OPT([-Wno-non-virtual-dtor])
    fi
    AC_CHECK_CC_OPT([-Wall])
