--- djvulibre-3.5.14/config/acinclude.m4.orig	2004-05-28 20:41:49.000000000 +0200
+++ djvulibre-3.5.14/config/acinclude.m4	2004-07-25 21:00:30.819537664 +0200
@@ -79,41 +79,6 @@
         [ac_debug=$enableval],[ac_debug=no])
    OPTS=
    AC_SUBST(OPTS)
-   saved_CXXFLAGS="$CXXFLAGS"
-   saved_CFLAGS="$CFLAGS"
-   CXXFLAGS=
-   CFLAGS=
-   for opt in $saved_CXXFLAGS ; do
-     case $opt in
-       -g*) test $ac_debug != no && OPTS="$OPTS $opt" ;;
-       -O*) ;;
-       *) CXXFLAGS="$CXXFLAGS $opt" ;;
-     esac
-   done
-   for opt in $saved_CFLAGS ; do
-     case $opt in
-       -O*|-g*) ;;
-       *) CFLAGS="$CFLAGS $opt" ;;
-     esac
-   done
-   if test x$ac_debug = xno ; then
-     OPTS=-DNDEBUG
-     AC_CHECK_CXX_OPT([-Wall],[OPTS="$OPTS -Wall"])
-     AC_CHECK_CXX_OPT([-O3],[OPTS="$OPTS -O3"],
-        [ AC_CHECK_CXX_OPT([-O2], [OPTS="$OPTS -O2"] ) ] )
-     dnl This triggers compiler bugs with gcc-3.2.2
-     dnl AC_CHECK_CXX_OPT([-funroll-loops], [OPTS="$OPTS -funroll-loops"])
-     cpu=`uname -m 2>/dev/null`
-     test -z "$cpu" && cpu=${host_cpu}
-     case "${host_cpu}" in
-        i?86)
-           opt="-mcpu=${host_cpu}"
-           AC_CHECK_CXX_OPT([$opt], [OPTS="$OPTS $opt"])
-           ;;
-      esac
-   else
-     AC_CHECK_CXX_OPT([-Wall],[OPTS="$OPTS -Wall"])
-   fi
    case x"$ac_debug" in
 changequote(<<, >>)dnl
      x[0-9])  OPTS="$OPTS -DDEBUGLVL=$ac_debug" ;;
