diff -aurN djvulibre-3.5.15.orig/gui/djview/Makefile.in djvulibre-3.5.15/gui/djview/Makefile.in
--- djvulibre-3.5.15.orig/gui/djview/Makefile.in	2004-08-24 19:58:55.000000000 +0200
+++ djvulibre-3.5.15/gui/djview/Makefile.in	2005-07-19 19:15:19.000000000 +0200
@@ -174,7 +174,7 @@
 install-bin: ${PROGRAM} FORCE
 	${INSTALL} -d ${DESTDIR}${bindir}
 	${LIBTOOL} --mode=install \
-	  ${INSTALL_PROGRAM} -s ${PROGRAM} ${DESTDIR}${bindir}
+	  ${INSTALL_PROGRAM} ${PROGRAM} ${DESTDIR}${bindir}
 
 install-man: FORCE
 	${INSTALL} -d ${DESTDIR}${mandir}/man1
diff -aurN djvulibre-3.5.15.orig/tools/Makefile.in djvulibre-3.5.15/tools/Makefile.in
--- djvulibre-3.5.15.orig/tools/Makefile.in	2005-03-13 20:17:32.000000000 +0100
+++ djvulibre-3.5.15/tools/Makefile.in	2005-07-19 19:15:38.000000000 +0200
@@ -71,7 +71,7 @@
 	${INSTALL} -d ${DESTDIR}${bindir}
 	for n in ${PROGRAMS} ; do \
 	  ${LIBTOOL} --mode=install \
-	    ${INSTALL_PROGRAM} -s $$n ${DESTDIR}${bindir} \
+	    ${INSTALL_PROGRAM} $$n ${DESTDIR}${bindir} \
           || exit ; done
 
 install-man: FORCE
