--- djvulibre-3.5.8/libdjvu/GException.cpp.orig	Tue Jul 30 05:50:42 2002
+++ djvulibre-3.5.8/libdjvu/GException.cpp	Fri Oct 11 00:57:50 2002
@@ -265,7 +265,7 @@
 #if !defined(WIN32) && !defined(__CYGWIN32__)
 // In general this is a bad idea since statics won't get initialized
 // under WINDOWS, even with CYGWIN.
-static void (*old_handler)() = set_new_handler(throw_memory_error);
+static void (*old_handler)() = std::set_new_handler(throw_memory_error);
 #endif // ! WIN32
 #endif // !_MSC_VER
 #endif // !NEED_DJVU_MEMORY
